src/CreateMiddleware/CreateMiddleware.php

```diff
@@ -41,10 +41,11 @@ public function process(ServerRequestInterface $request, DelegateInterface $dele
     /**
      * @param string $class
      * @param string|null $projectRoot
+     * @param string|null $classSkeleton
      * @return string
      * @throws CreateMiddlewareException
      */
-    public function process($class, $projectRoot = null)
+    public function process($class, $projectRoot = null, $classSkeleton = null)
```

Comment:
Ok, but if you change `self::` to `static::` you don't need to change signature so you are not creating possible BC Break.